### PR TITLE
UMD Fix

### PR DESCRIPTION
**Pull request by GordonSmith**
Clean up and fix UMD stubs.

Signed-off-by: Gordon Smith <gordon.smith@lexisnexis.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@buunguyen Please Review

File: widgets/src/c3/Area.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
     if (typeof define === "function" && define.amd) {
         define(["./CommonND"], factory);
     } else {
-        root.Area = factory(root.CommonND);
+        root.C3_Area = factory(root.C3_CommonND);
```
Comment (GordonSmith) [inline widgets/src/c3/Area.js]: Should be c3_Area (lower case "c")

File: widgets/src/c3/Common.js [hunk 2]
```diff
@@ -3,7 +3,7 @@
     if (typeof define === "function" && define.amd) {
         define(["d3/d3", "c3/c3", "../common/HTMLWidget", "css!c3/c3"], factory);
     } else {
-        root.Pie = factory(root.d3, root.c3, root.HTMLWidget);
+        root.C3_Common = factory(root.d3, root.c3, root.HTMLWidget);
```
Comment (GordonSmith) [inline widgets/src/c3/Common.js]: should be root.Common_HTMLWidget?

Comment (buunguyen) [inline widgets/src/c3/Common.js]: That's an old commit. The most recent commit should be identical to what I have in https://github.com/hmsonline/Visualization. Everything was changed lower case.

**Comment by GordonSmith** [comment]
I suspect not prefixing the widgets in the common folder with "Common_" will potentially cause issue when it comes to writing the resolver and further down the line in namespacing.  We could special case it in the root.require, but probably better to make it consistent?

**Comment by buunguyen** [comment]
@GordonSmith you've been commenting on old commits that have been refactored several times. In HEAD, all should have prefixes and all prefixes have lowercase.

**Comment by GordonSmith** [comment]
@buunguyen Should be correct now.

**Comment by GordonSmith** [comment]
@buunguyen - This looks ok to me now.

**Comment by buunguyen** [comment]
LGTM.

**Pull request merged**